packages/frontend/amp/components/AdConsent.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { AMPJSON } from './AMPJSON';
+import { css, cx } from 'emotion';
+import Tick from '@guardian/pasteup/icons/tick.svg';
+
+const fontFamily =
+    'GuardianTextSans,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif';
+
+const consentUIStyle = css`
+    font-family: ${fontFamily};
+    font-size: 1rem;
+    line-height: 1.25rem;
+    color: #f6f6f6;
+    background-color: rgb(51, 51, 51);
```

Comment:
It would be awesome if we could use colours from [the Pasteup palette](https://github.com/guardian/dotcom-rendering/tree/master/packages/pasteup#palette)

Reply:
Amazin!